tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/imports/ImportsTest.java

```diff
@@ -63,4 +63,10 @@ public void testSameModuleNameDifferentOrgImports() {
         BValue[] returns = BRunUtil.invoke(result, "getStringValueOfPI");
         Assert.assertTrue((returns[0]).stringValue().startsWith("3.14"));
     }
+
+    @Test(description = "Test auto imports")
+    public void testPredeclaredModules() {
+        BCompileUtil.compile("test-src/imports/predeclared-imports", "bar");
```

Comment:
Are we only building these or are we also validating the functionality?

Reply:
Only building that module. Is it need to validate functionality here?

Reply:
Could you please send a separate PR validating 0 diagnostics.